File: 2022/02/t.py
cal = open('input.txt', 'r', encoding='utf-8').read().strip().split('\n')

soma = 0
for txt in cal:
    b1 = ord(txt[0]) - ord('A')
    b2 = ord(txt[2]) - ord('X')
    if b1 == b2:
        y = 3
    elif b1 == (b2 + 2) % 3:
        y = 6
    else:
        y = 0
    soma += (b2 + 1) + y
    # print(b, x, y)
print('Round 1:', soma)  # 12794

soma = 0
for txt in cal:
    b1 = ord(txt[0]) - ord('A')
    b2 = ord(txt[2]) - ord('X')

    x = (b1 + b2) % 3
    if x == 0:
        x = 3

    soma += x + (b2 * 3)

print('Round 2:', soma)  # 14979

File: 2022/02/t.java
import java.io.IOException;
import java.nio.charset.StandardCharsets;
import java.nio.file.Files;
import java.nio.file.Paths;

public class t {
    public static void main(String[] args) throws IOException {
        String[] content = Files.readString(Paths.get("input.txt"), StandardCharsets.UTF_8).strip().split("\r?\n");

        int soma = 0;
        int match = 0;
        for (String txt : content) {
            int b1 = (txt.charAt(0) - 'A');
            int b2 = (txt.charAt(2) - 'X');

            if (b1 == b2)
                match = 3;
            else if (b1 == (((b2 + 2) % 3)))
                match = 6;
            else
                match = 0;
            soma += (b2 + 1) + match;
        }
        System.out.printf("Round 1: %s%n", soma);

        soma = 0;
        for (String txt : content) {
            int b1 = (txt.charAt(0) - 'A');
            int b2 = (txt.charAt(2) - 'X');

            match = (b1 + b2) % 3;
            if (match == 0)
                match = 3;

            soma += match + (b2 * 3);
        }
        System.out.printf("Round 2: %s%n", soma);
    }
}
